gridstatus/eia.py

```diff
@@ -727,6 +752,28 @@ def _handle_fuel_type_data(df):
     return df
 
 
+def _handle_henry_hub_natural_gas_spot_prices(df):
+    df["Interval Start"] = pd.to_datetime(df["period"], utc=True)
```

Comment:
are you sure this is UTC time?

Reply:
The `period` isn't a time, it's only a date. But, we don't really support datasets that only have a date column so I parsed it into UTC.